sso/test-steps/src/main/java/org/eclipse/kapua/service/sso/steps/SsoServiceSteps.java

```diff
@@ -0,0 +1,211 @@
+/*******************************************************************************
```

Comment:
Please change complete header to: 
/*******************************************************************************
 * Copyright (c) 2020 Eurotech and/or its affiliates and others
 *
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 *
 * Contributors:
 *     Eurotech - initial API and implementation
 *******************************************************************************/